From d20195c84f5efa4a61588906b7478e5cd3ac9ab0 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Sun, 26 Aug 2018 12:11:21 -0700
Subject: [PATCH] sepolicy: Update to match new qcom sepolicy

persist_block_device is now defined in device/qcom/sepolicy
and /data/vendor/display has been relabeled

Change-Id: I330fdd37bc7a6693f51f5cbe6b71189401eec316
---
 qcom/common/device.te                  | 1 -
 qcom/common/hal_livedisplay_default.te | 4 ++--
 2 files changed, 2 insertions(+), 3 deletions(-)
 delete mode 100644 qcom/common/device.te

diff --git a/qcom/common/device.te b/qcom/common/device.te
deleted file mode 100644
index 9e49627..0000000
--- a/qcom/common/device.te
+++ /dev/null
@@ -1 +0,0 @@
-type persist_block_device, dev_type;
diff --git a/qcom/common/hal_livedisplay_default.te b/qcom/common/hal_livedisplay_default.te
index 3c2dacc..023b095 100644
--- a/qcom/common/hal_livedisplay_default.te
+++ b/qcom/common/hal_livedisplay_default.te
@@ -4,8 +4,8 @@ allow hal_livedisplay_default qdisplay_service:service_manager find;
 binder_call(hal_livedisplay_default, hal_graphics_composer_default)
 
 # Allow LiveDisplay to store files under /data/vendor/display and access them
-allow hal_livedisplay_default display_misc_file:dir rw_dir_perms;
-allow hal_livedisplay_default display_misc_file:file create_file_perms;
+allow hal_livedisplay_default display_vendor_data_file:dir rw_dir_perms;
+allow hal_livedisplay_default display_vendor_data_file:file create_file_perms;
 
 # Allow LiveDisplay to access pps socket
 unix_socket_connect(hal_livedisplay_default, pps, mm-pp-daemon)
-- 
2.17.1

